sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1226,6 +1226,28 @@ pi_result piGetDeviceAndHostTimer(pi_device Device, uint64_t *DeviceTime,
   return pi2ur::piGetDeviceAndHostTimer(Device, DeviceTime, HostTime);
 }
 
+pi_result piextEnablePeerAccess(pi_device command_device,
```

Comment:
@kbenzie : should we get any new changes into the adapters code in intel/llvm to follow UR convention (camelCase) so that way when moving the code to the UR repo we have fewer issues, or will we take care of that when time comes?

Reply:
For the PI layer I'm not too fussed about naming conventions since the code will go be removed but it makes sense for the UR adapter implementations.

Reply:
Yeah I followed UR convention in all UR stuff, and the existing pi variable snake_case in pi stuff, that will eventually just be removed.

Reply:
thanks @kbenzie , @JackAKirk